sentry-opentelemetry/sentry-opentelemetry-agent/README.md

```diff
@@ -44,3 +44,25 @@ To enable debug logging for Sentry, please provide `SENTRY_DEBUG=true` as enviro
 add `debug=true` to your `sentry.properties`.
 
 To also show debug output for OpenTelemetry please add `-Dotel.javaagent.debug=true` to the command.
+
+## Getting rid of exporter error messages
```

Comment:
should we have something in `sentry.properties` for that as well?

Reply:
It's basic OTEL config. so I guess it's not worth bloating our config with it.